packages/contracts-bedrock/scripts/deploy/DeployDelayedWETH.s.sol

```diff
@@ -45,6 +45,9 @@ contract DeployDelayedWETHInput is BaseDeployIO {
         } else if (_sel == this.delayedWethOwner.selector) {
             require(_value != address(0), "DeployDelayedWETH: delayedWethOwner cannot be zero address");
             _delayedWethOwner = _value;
+        } else if (_sel == this.delayedWethImpl.selector) {
+            require(_value != address(0), "DeployDelayedWETH: delayedWethImpl cannot be zero address");
```

Comment:
But delayedWethImpl should be a 0 value to deploy the initial implementation. I think the assertion is being skipped anyway because this gets shimmed which is why it works but probably should just remove the require.